specification/_global/exists/examples/xCodeSamples/DocumentExistsRubyExample1.yaml

```diff
@@ -0,0 +1,8 @@
+- lang: Ruby
+  # label: 
+  source: |
+    response = client.exists(
+      index: 'my-index-000001',
+      id: 0
+    )
+    puts response
```

Comment:
Not sure if that matters, but this file is not intended like the others.

Reply:
Ah, good catch, that's why the preview was failing :)